src/main/java/org/intellij/sdk/codesync/database/Database.java

```diff
@@ -0,0 +1,103 @@
+package org.intellij.sdk.codesync.database;
+
+import com.intellij.openapi.application.ApplicationManager;
```

Comment:
Unused import. Kindly double check all over the PR for unused imports @syedgulahmed